block_designs.tex

```diff
@@ -1,6 +1,144 @@
 \chapter{Blokové plány}
 
+Blokové plány sú kombinatorické štruktúry, pre ktoré existuje veľa využití v štatistike, dizajne experimentov, teórii kódovania, teórii grafov a ďalších oblastiach. Napríklad aj pri dizajne spoločenských hier :) Populárna hra Dobble je vlastne blokový plán, kde jednotlivé karty sú bloky a symboly na kartách sú objekty.
+
+\section{Symetrické blokové plány}
+
+\begin{definition}
+
+Nech $X = \set{x_1, \ldots, x_v}$ je množina objektov a $\mathcal{B} = \set{X_1, \ldots, X_v} \subset \powerset(X)$ je množina podmnožín objektov (tieto podmnožiny voláme \emph{bloky}), pričom sú splnené nasledujúce podmienky:
+\begin{enumerate}
+	\item $|X_i| = k$ pre $i = 1, 2, \ldots, v$
+	\item $|X_i \cap X_j| = \lambda$ pre $i \neq j$
+	\item $0 < \lambda < k < v - 1$
+\end{enumerate}
+Potom systém blokov $\mathcal{B}$ voláme $(v, k, \lambda) - konfigurácia$ (alebo symetrický blokový plán).
```

Comment:
Dolars should be used for math, not for italits. For this purpose, use `\emph{$(v, k, \lambda)$-konfigurácie}`

Reply:
Removed italics from "(v, k, \lambda) - konfigurácia" notation everywhere apart definition to be in accordance with "(v, k, \lambda)-diferenčnou množinou" notation.